x-pack/plugins/siem/public/store/inputs/reducer.ts

```diff
@@ -33,48 +35,60 @@ export const initialInputsState: InputsState = {
     query: [],
     policy: {
       kind: 'manual',
-      duration: 5000,
+      duration: 300000,
```

Comment:
Is this correct as `300000` (5min), or should it be `30000` (30sec)? Same as on line `53` below.

Reply:
the default is five minutes, if I forget to change the one below I will do it.